### PR TITLE
Register Subdomain: gabriel.almeida

### PR DESCRIPTION
**Pull request by gabriellukke**
Added the domain: gabriel.almeida

### CODE REVIEW

File: domains/gabriel.almeida.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "gabriellukke",
+        "email": "gabriellukke@outlook.com"
+      },
+      "record": {
+        "URL": ""gabriellukke.github.io""
```
Comment (WilliamDavidHarrison) [inline domains/gabriel.almeida.json:8]: Please remove the extra quotations and add `https://` infront of the URL.

**Review by wdhdev — CHANGES_REQUESTED** [review]
You need to register the domain `almeida.is-a.dev` before you can get a nested subdomain.

Please also refer to the review comments below.

File: domains/gabrielalmeida.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "gabriellukke",
+        "email": "gabriel.dev.almeida@outlook.com"
+      },
+      "record": {
+        "URL": "https://gabriellukke.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/gabrielalmeida.json:8]: Please create a website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by TheBotlyNoob** [comment]
I'm closing this because of the large amounts of pull requests. You can make another PR later with fixes if you want.

**Pull request closed without merging**